web/src/HUD.tsx

```diff
@@ -89,7 +89,11 @@ class HUD extends Component<HudProps, HudState> {
   componentWillMount() {
     incr("ui.web.init", { ua: window.navigator.userAgent })
     this.unlisten = this.history.listen((location, _) => {
-      let tags = { path: location.pathname }
+      let isLog =
+        location.pathname.startsWith("/r/") &&
+        !location.pathname.endsWith("/preview")
+      let isPreview = location.pathname.endsWith("/preview")
+      let tags = { isLog: isLog.toString(), isPreview: isPreview.toString() }
```

Comment:
would it make sense to add a digest if it's available, like we do for other fields? https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest

Reply:
In the future yeah but for now I didn't care about the path names for their own sake I just wanted to be able to tell what kind of paths they were.